webpack.prod.config.js

```diff
@@ -1,98 +1,77 @@
   devtool: false,
+  resolve: {
+    extensions: ['.js', '.scss'],
+  },
   entry: {
-    main: './src/index.js',
+    main: './index.js',
     vendor: [
-      'babel-polyfill', 'react', 'react-dom', 'react-hot-loader', 'react-redux',
+      'babel-polyfill', 'react', 'react-dom', 'react-redux',
       'react-router', 'react-router-redux', 'redux', 'redux-action', 'redux-saga',
     ],
   },
   output: {
-    path: buildPath,
-    publicPath: '',
+    path: resolve(__dirname, 'build'),
+    publicPath: '/',
     filename: '[name].[chunkhash:6].js',
     chunkFilename: '[name].[chunkhash:6].[ext]',
   },
   module: {
     loaders: [
       {
-        test: /\.js(x?)$/,
-        loader: 'babel',
-        exclude: [nodeModulesDir],
-      },
-      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+        exclude: /node_modules/,
+      }, {
         test: /\.scss$/,
         loaders: [
-          'style',
-          'css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]',
-          'postcss',
-          'sass',
+          'style-loader',
+          'css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]',
+          'sass-loader',
         ],
-      },
-      {
+      }, {
         test: /\.(png|jpg|jpeg|gif|svg|woff|woff2|eot)$/,
-        loader: 'url',
-        include: [
-          path.resolve(__dirname, 'src/assets'),
-        ],
+        loader: 'url-loader',
         query: {
           context: './src/assets/',
-          name: [buildAssetsFolderName, '[path][name].[hash:base64:6].[ext]'].join('/'),
+          name: ['assets', '[path][name].[hash:base64:6].[ext]'].join('/'),
           limit: 500,
         },
-      },
-      {
-        // this one is for loading vendor assets
-        test: /\.(png|jpg|jpeg|gif|svg|woff|woff2|eot)$/,
-        loader: 'url',
-        include: [
-          path.resolve(nodeModulesDir, 'vis/dist/'),
-          path.resolve(nodeModulesDir, 'video.js/dist/'),
+      }, {
+        test: /\.css$/,
+        loaders: [
+          'style-loader',
+          'css-loader',
         ],
-        query: {
-          context: nodeModulesDir,
-          name: [buildAssetsFolderName, 'vendor', '[path][name].[hash:base64:6].[ext]'].join('/'),
-          limit: 500,
-        },
       },
     ],
   },
-  postcss: [
-    require('autoprefixer'),
-    require('precss'),
-  ],
   plugins: [
-    ignore,
     new webpack.DefinePlugin(GLOBALS),
     new WebpackMd5Hash(),
     new ExtractTextPlugin('main.[contenthash:6].css'),
     new webpack.optimize.CommonsChunkPlugin({
       names: ['vendor'],
     }),
-    new webpack.optimize.DedupePlugin(),
     new webpack.optimize.OccurrenceOrderPlugin(),
     new webpack.optimize.UglifyJsPlugin({
       sourceMap: false,
       compress: {
         warnings: false,
-      }
+      },
     }),
     new HtmlWebpackPlugin({
-      template: './src/index.html',
+      template: './index.html',
     }),
   ]
 };
```

Comment:
When I looked at this file, I found that this file is not very similiar to webpack.config.js. Ideally they should be vary similar. E.g. this sets `` path: resolve(__dirname, 'build'), `` while the other sets `` path: resolve(__dirname, 'dist'), ``